src/utils/nft-metadata.ts

```diff
@@ -22,6 +22,28 @@ export const findUsedTrait = (jsonMetadata: JsonMetadata) =>
 export const findSignedTrait = (jsonMetadata: JsonMetadata) =>
   findTrait(jsonMetadata, SIGNED_TRAIT).value === 'true';
 
+export const findRarityTrait = (jsonMetadata: JsonMetadata) =>
+  findTrait(jsonMetadata, RARITY_TRAIT).value;
+
 export const fetchOffChainMetadata = async (uri: string) => {
   return (await axios.get<JsonMetadata>(uri)).data;
 };
+
+export function generateComicAttributes(): [boolean, boolean][] {
+  const combinations: [boolean, boolean][] = [];
+
+  for (const i of [false, true]) {
+    for (const j of [false, true]) {
+      combinations.push([i, j]);
+    }
+  }
```

Comment:
This is the single silliest piece of code I have seen this year. I love it, it's amazing. I want to screenshot it and put it on my wall.

BUT... why not simply make an array of [[false,false],[false,true],[true,false],[true,true]] ?

Reply:
haha, didn't want to write a hard coded var